Gemfile

```diff
@@ -1,3 +1,4 @@
 source 'https://rubygems.org'
 
 gemspec :name => RUBY_PLATFORM == 'java' ? 'gollum-lib_java' : 'gollum-lib'
+gem "gollum-rugged_adapter"
```

Comment:
I wanted to switch to `rugged` in order to make sure that everything would work without the ws subs. I'll check if the tests pass with the `grit` adapter in the end, though. If not, shall I move this dependency to the gemspec instead?